### ISSUE TITLE
Zoom-Button

### ISSUE THREAD

**Issue by Nokius** [tunp/monavsailfish#4]
I found today the zoom-Button the text is to small

May use the Zoom Icon from images/oxygen with http://doc.qt.io/qt-5/qml-qtquick-controls-button.html#iconSource-prop

Or use http://doc.qt.io/qt-5/qml-qtquick-controls-styles-buttonstyle.html
It should work like this https://gist.github.com/Nokius/8963d054a69d5074288c 
you need to install Qt.controls to use it on the device

**Comment by tunp** [comment]
Thats true. Images from oxygen could be used. But I'm not sure if we need Qt controls button for this. Would something like this be enough? https://gist.github.com/tunp/f55e1fd19e40b4d71571
Thanks.

**Comment by Nokius** [comment]
I had no luck using it the SDK :-/
